TTYC.Application/Carts/GetCart/GetCartHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿using MediatR;
+using Microsoft.EntityFrameworkCore;
+using TTYC.Application.Interfaces;
+using TTYC.Application.Models;
+using TTYC.Persistence;
+
+namespace TTYC.Application.Carts.GetCart
+{
+    public class GetCartHandler : IRequestHandler<GetCartQuery, ViewCart>
+    {
+        private readonly ApplicationDbContext dbContext;
+        private readonly ICurrentUserService currentUserService;
+
+        public GetCartHandler(ApplicationDbContext dbContext, ICurrentUserService currentUserService)
+        {
+            this.dbContext = dbContext;
+            this.currentUserService = currentUserService;
+        }
+
+        public async Task<ViewCart> Handle(GetCartQuery query, CancellationToken cancellationToken)
+        {
+            var cart = new ViewCart
+            {
+                CartItems = await dbContext.CartItems
+                .Include(x => x.Product)
+                .Where(x => x.CartId == currentUserService.UserId)
+                .ToListAsync(cancellationToken),
+
+
+                TotalSum = (from items in dbContext.CartItems
+                            where items.CartId == currentUserService.UserId
```

Comment:
You use Fluent syntax everywhere and now you use query syntax where Fluent is possible. I think it is better to keep to one of them